packages/api/controllers/auth.js

```diff
@@ -80,36 +75,41 @@ const markAsRead = function (messageId, gmail) {
     }
   });
 };
-
 async function getNewDocs(auth) {
   const gmail = google.gmail({ version: 'v1', auth });
-  let query = '';
+  const query = '';
   const nomNames = await getAwaitingHipaa();
-
-  if (!nomNames) {
-    console.log('No Nominations Currently Awaiting HIPAA');
-  } else {
-    nomNames.forEach((nomination) => query += `subject:'${nomination}'`);
-    query += 'AND is:unread';
-    gmail.users.messages.list({
-      userId: 'me',
-      q: `{${query}}`,
-    }, (err, res) => {
-      if (err) return console.log('The API returned an error: ', err);
-      const { messages } = res.data;
-      if (!messages) {
-        console.log('no messages');
-      } else if (messages.length) {
-        const messageIds = [];
-        messagesAndAttachments(messages, messageIds, gmail, nomNames);
-      }
-    });
+  const receiptNoms = await getVerifiedNoms();
+  if (Object.entries(nomNames).length) {
+    getNewAttachments(nomNames, gmail, query);
+  }
+  const receipts = Object.entries(receiptNoms);
+  if (receipts) {
+    getNewAttachments(receiptNoms, gmail, query);
   }
 }
-
-const messagesAndAttachments = (messages, messageIds, gmail, nomNames) => {
+const getNewAttachments = (nominations, gmail, query) => {
+  const nomNames = Object.keys(nominations);
+  nomNames.forEach((nomination) => query += `subject:'${nomination}'`);
```

Comment:
Does this need to have a space between each value or something? Right now if `nomNames` has more than one value it looks like it will generate this: `subject:'a'subject:'b'subject:'c'` which seems off.

Reply:
We're making separate queries to Google for each nomination that fits the specific filter. Because there is another nomNames in a different function, I will change this out for a different name to make it less confusing.